css/css-content/content-animation.html

```diff
@@ -23,29 +27,39 @@
 
 </style>
 <body>
-<div class="target"></div>
 <script>
 
-test(function() {
-  const target = document.querySelector(".target");
-  const style = getComputedStyle(target, "::after");
+const contentAnimationTest = (className, expectedInitialValue, title) => {
+    test(() => {
+        const target = document.body.appendChild(document.createElement("div"));
+        target.classList.add("target");
+        if (className)
+            target.classList.add(className);
+
+        const style = getComputedStyle(target, "::after");
 
-  assert_equals(style.content, '"default"', "Before the animation is applied.");
+        assert_equals(style.content, expectedInitialValue, "Before the animation is applied.");
 
-  target.classList.add("animated");
-  const animation = target.getAnimations({ subtree: true })[0];
+        target.classList.add("animated");
+        const animation = target.getAnimations({ subtree: true })[0];
 
-  const testContentAtTime = (time, expected) => {
-    animation.currentTime = time;
-    assert_equals(style.content, `"${expected}"`, `Check the animated value at time = ${time}ms`);
-  };
+        const testContentAtTime = (time, expected) => {
+          animation.currentTime = time;
+          assert_equals(style.content, `"${expected}"`, `Check the animated value at time = ${time}ms`);
+        };
+
+        testContentAtTime(0, 'from');
+        testContentAtTime(499, 'from');
+        testContentAtTime(500, 'to');
+        testContentAtTime(999, 'to');
+        testContentAtTime(1000, 'to');
```

Comment:
nit: can you be consistent between single & double quotes in this test? I prefer double quotes personally, and backticks to escape double quotes.